DataFormats/HeavyIonEvent/src/EvtPlane.cc

```diff
@@ -22,22 +23,37 @@
 
 
 //
+// $Id: EvtPlane.cc,v 1.4 2009/09/08 12:33:12 edwenger Exp $
 //
 
 #include "DataFormats/HeavyIonEvent/interface/EvtPlane.h"
 
 using namespace reco;
-
-EvtPlane::EvtPlane(double planeA,double sumSin, double sumCos, std::string label)
+using namespace std;
+EvtPlane::EvtPlane(int epindx, int level, double planeA,double sumSin, double sumCos, double sumw, double sumw2, double sumPtOrEt, double sumPtOrEt2, uint mult)
   : 
-   angle_(planeA),
-   sumSin_(sumSin),
-   sumCos_(sumCos),
-   label_(label)
+   indx_(epindx),
+   sumw_(sumw),
+   sumw2_(sumw2),
+   sumPtOrEt_(sumPtOrEt),
+   sumPtOrEt2_(sumPtOrEt2),
+   mult_(mult)
 {
+  angle_[level] = planeA;
```

Comment:
unless intentional, initialize all other values to 0.
Currently other 3 (or 2) values are uninitialized and can lead to unpredictable behavior.